src/components/Wizard/Wizard.test.tsx

```diff
@@ -126,28 +163,88 @@ describe('<Wizard />', () => {
 		},
 	);
 
-	test('Should not submit when invalid data', () => {
+	test('should go to navigate to previous step when clicking previousbutton', () => {
+		const pushSpy = jest.fn();
+		(useHistory as jest.Mock).mockReturnValue({
+			push: pushSpy,
+		});
 		renderComponent(
 			{
 				file: mockFile,
-				metadata: { documentDescription: '', executionDate: '' },
+				metadata: { mockData },
 			},
 			'/step2',
 		);
-		fireEvent.click(screen.getByText('Opslaan'));
-		expect(onMetadataSubmitMock).not.toHaveBeenCalled();
+		const previousButton = screen.getByTestId('previous-button');
+		expect(screen.getByTestId('previous-button')).toBeInTheDocument();
+		fireEvent.click(previousButton);
+
+		expect(pushSpy).toHaveBeenCalledWith('/');
 	});
+	describe('onMetadataSubmit()', () => {
+		test('is called when saving while <MetaDataForm /> is valid', async () => {
+			const pushSpy = jest.fn();
+			const setMetadataSpy = jest.spyOn(actions, 'setMetadata');
+			useHistoryMock.mockReturnValue({ push: pushSpy } as any);
+			const metadata: MetadataExample = {
+				documentDescription: '__DOCUMENT_DESCRIPTION__',
+				executionDate: '__EXECUTION_DATE__',
+			};
+			getMetadataFromStoreMock.mockReturnValue(metadata);
+			renderComponent({ file: mockFile, metadata: {} }, '/step2');
+			const { onChange } = mockComponentProps<ComponentProps<typeof MetadataForm>>(MetadataFormMock);
+			act(() => {
+				onChange(metadata, true, []);
+			});
+			act(() => {
+				fireEvent.click(screen.getByText('Opslaan'));
+			});
+			await waitFor(() => {
+				expect(setMetadataSpy).toHaveBeenCalledWith(metadata);
+				expect(onMetadataSubmitMock).toHaveBeenCalledWith({
+					file: mockFile,
+					metadata,
+				});
+				expect(pushSpy).toHaveBeenCalledWith('/');
+			});
+		});
+
+		test('is NOT called when saving while <MetaDataForm /> is NOT valid', async () => {
+			const pushSpy = jest.fn();
+			useHistoryMock.mockReturnValue({ push: pushSpy } as any);
+			renderComponent({ file: mockFile, metadata: {} }, '/step2');
+
+			act(() => {
+				fireEvent.click(screen.getByText('Opslaan'));
+			});
+			await waitFor(() => {
+				expect(onMetadataSubmitMock).not.toHaveBeenCalled();
+			});
+		});
 
-	// test('Should submit when valid data', () => {
-	// not working yet
-	// renderComponent(storeState, '/step2', { formValues: mockData, isValidForm: true });
-	// // const spy = jest.spyOn(selectors, 'getFileFromStore');
-	// // expect(spy).toHaveBeenCalled();
-	// expect(screen.getByText('Vorige')).toBeInTheDocument();
-	// // console.log(container);
-	// expect(onMetadataSubmitMock).toHaveBeenCalled();
-	// fireEvent.click(screen.getByText('Opslaan'));
-	// fireEvent.click(screen.getByText('Vorige'));
-	// expect(screen.queryByTestId('step-1')).toBeInTheDocument();
-	// });
+		test('should be able to catch error on submit', async () => {
```

Comment:
I am seeing this on the console:
![image](https://user-images.githubusercontent.com/1095870/145773994-ae004db1-5e47-4e2a-b1c1-76be239030ba.png)

I don't think you need to spy on the console.log. Instead you can catch an error in jest like so:
```typescript
const t = () => {
    // Trigger some code that throws an exception
};

// @see https://jestjs.io/docs/expect#tothrowerror
expect(t).toThrowError(); // Optionally you can assert that a specific error or error with a specific message was thrown.
```